t/distpl.t

```diff
@@ -0,0 +1,16 @@
+use strict;
+use warnings;
+use Test::More 0.88;
+
+use ExtUtils::Manifest 'maniread';
```

Comment:
is this needed?

Reply:
mm, no, I just cribbed the code semi-blindly from another test and then cut it down till it worked, clearly didn't cut it enough. :)